src/_posts/languages/php/2000-01-01-start.md

```diff
@@ -123,9 +123,9 @@ the used formula is: `floor(available_memory / php_memory_limit) + 2`
 
 {: .table }
 | Container Size | Default Concurrency |
-| -------------- | ------------------- |
+| -------------: | ------------------- |
```

Comment:
issue: I don't think that's intended, is it?
```suggestion
| -------------- | ------------------- |
```

Reply:
It is :) It allows values in the column to be right aligned.

Reply:
OH WOW :star_struck: